src/ReactWrapperComponent.jsx

```diff
@@ -54,7 +54,7 @@ export default function createWrapperComponent(node, options = {}) {
       const component = this._reactInternalInstance._renderedComponent;
       const inst = component.getPublicInstance();
       if (inst === null) {
-        return component;
+        return component._instance;
```

Comment:
Thinking about it more, it seems like React doesn't want you to be able to call `findDOMNode` on SFCs so the alternative might just be documenting that those methods don't work on SFCs.